### ISSUE TITLE
Number in configuration page is weird.

### ISSUE THREAD

**Issue by Nick-Hopps** [yuzu-emu/yuzu#8698]
System Info:  Windows 11
yuzu version:  EA 2872

<img width="397" alt="Snipaste_2022-08-01_12-25-28" src="https://user-images.githubusercontent.com/16911282/182072182-e9a718a2-4844-4ccf-8fa5-155d61216d31.png">

<img width="609" alt="Snipaste_2022-08-01_12-25-42" src="https://user-images.githubusercontent.com/16911282/182072193-8fa22558-0994-439b-95df-e42099265403.png">

**Comment by german77** [comment]
Does it happen on mainline as well. That doesn't look normal at all

**Comment by Docteh** [comment]
Few assorted questions, is that in wine or actual windows?

What version of windows is that, and what language is it set to? Seems like yuzu is being forced to use a different font than on most screenshots I've seen. 

In the menu I recognize the font sizing as what is sometimes referred to as half width characters. So like a Chinese character would be the full width.

Also if you were to open like notepad, is it the same font as "limit speed percent", what is the name of the font? Also check for Arial

I've learned my lesson on changing locale on main os. So I might install evaluation copy of the language you're using in virtual machine.

**Comment by Docteh** [comment]
I can rule Japanese out.

Windows 11 Japanese, set yuzu UI to English. yuzu 1118 mainline
![image](https://user-images.githubusercontent.com/190571/182095563-ab96916e-e8a4-4bea-9814-fcfcf4b38bf8.png)

![image](https://user-images.githubusercontent.com/190571/182095152-fcd095b9-7ca8-406a-bb01-073865d82660.png)

Acknowledging this, not planning on touching this path thing though.
![image](https://user-images.githubusercontent.com/190571/182096379-48001513-67a0-47f1-85e1-926be864041f.png)

**Comment by Nick-Hopps** [comment]
My system language is Chinese, but seems not related since I've changed system language to English and is still the same.

**Comment by Docteh** [comment]
Which language should I choose for ISO?

I switched the Japanese install to Chinese, whatever the last option is, and most things pick up the language change except for the default font that Qt uses, and the interesting replacement of showing yen symbol instead of \ in the log window. 

So changing from Japanese to Chinese wasn't 100%

**Comment by Nick-Hopps** [comment]
It's exactly what it shows: 〈〇〇%

**Comment by Nick-Hopps** [comment]
as I change the system format to English, everything's fine now.

![image](https://user-images.githubusercontent.com/16911282/182161653-de6a4b99-984e-4711-ad56-fa9a3ad5eba8.png)

**Comment by Nick-Hopps** [comment]
I guess maybe is that Qt not applying the right font for responding language format of Date and Number.

**Comment by Docteh** [comment]
Since those characters show like that in GitHub and discord, not a font. Interesting localization. I'll investigate.

**Comment by Nick-Hopps** [comment]
Not familiar with Qt so waiting for your good news. 😄

**Comment by Nick-Hopps** [comment]
I found it work well in windows 10, perhaps a bug of window 11?

**Comment by Docteh** [comment]
English locale for widget
![image](https://user-images.githubusercontent.com/190571/182258334-d9ffef26-884e-4a2b-8d9e-a15df789ecbe.png)

Chinese locale for widget - this difference only shows on Windows 11, possibly influenced by the japanese install?
`ui->speed_limit->setLocale(QLocale::Chinese);`
![image](https://user-images.githubusercontent.com/190571/182258474-730ea52b-2e81-4906-be4a-137516228938.png)

Solution to `\` being shown as `¥` looks to be to not use `MS UI Gothic` as a font. 
![image](https://user-images.githubusercontent.com/190571/182264566-7da6130e-d9a3-4625-b77c-d42265c2a7c1.png)

Default font that Qt picks up is still Japanese, even after explicitly telling windows to use Chinese for non-unicode apps, and to use English for the rest. 
![image](https://user-images.githubusercontent.com/190571/182271116-12e0fa66-decb-4fe3-8e7d-bf54620043e4.png)

Next step looks to be to fresh install with a Chinese ISO
![image](https://user-images.githubusercontent.com/190571/182270330-cd2cfb08-34e0-4ce6-9e14-9c1ca903fb89.png)

**Comment by Docteh** [comment]
Chinese Traditional

`<System>`
![image](https://user-images.githubusercontent.com/190571/182351712-d14a4f0c-93ed-45db-af6a-95f0b1c659b1.png)
English
![image](https://user-images.githubusercontent.com/190571/182351394-b63787c8-6681-4f32-9042-72e461ee90ef.png)

![image](https://user-images.githubusercontent.com/190571/182355579-ba6a5076-f631-44cf-9490-ae97dd4a50f4.png)

Found a bug that explicitly mentions Chinese Simplified. 
https://bugreports.qt.io/browse/QTBUG-99552

Going to try changing from Chinese Traditional to Chinese Simplified, but maybe I need to install a third VM to reproduce.

**Comment by Docteh** [comment]
![image](https://user-images.githubusercontent.com/190571/182480898-d50224eb-f41c-4e70-956b-703ed0e0f247.png)
SimSun,9

I was hoping I'd install the right language of Windows 11 and get garbled numbers, no dice.

```
The hex is the UTF-8 byte sequence
0 〇 0xe3 0x80 0x87
1 〈 0xe3 0x80 0x88
2 〉 0xe3 0x80 0x89
3 《 0xe3 0x80 0x8a
4 》 0xe3 0x80 0x8b
5 「 0xe3 0x80 0x8c
6 」 0xe3 0x80 0x8d
7 『 0xe3 0x80 0x8e
8 』 0xe3 0x80 0x8f
9 【 0xe3 0x80 0x90
0 】 0xe3 0x80 0x91
5 〒 0xe3 0x80 0x92
6 〓 0xe3 0x80 0x93
7 〔 0xe3 0x80 0x94
8 〕 0xe3 0x80 0x95
9 〖 0xe3 0x80 0x96
```
14.9 GB?

Looks like itoa but instead of starting at 0x30 it accidentally starts at 0x3007 IDEOGRAPHIC NUMBER ZERO

**Comment by Nick-Hopps** [comment]
yes it's 14.9, and another is 100, seems like the case

**Comment by Docteh** [comment]
https://www.qt.io/blog/teaching-qlocale-more-about-number-formats
https://bugreports.qt.io/browse/QTBUG-85409
https://en.wikipedia.org/wiki/Suzhou_numerals


https://github.com/qt/qtbase/commit/d8538163075a2058f110bc77e7d481ef1ab7612c Oh since this'll cross reference, looks like this fix wasn't backported to 5.15.x  😛 
Looks like their fix is in for v6.0.0-alpha1, so any 6.x version we choose will have this fix.

```
3007 〇
3021 〡
3022 〢
3023 〣
3024 〤
3025 〥
3026 〦
3027 〧
3028 〨
3029 〩
```
Looks like it was going for 〡〇〇%
〥〇%
〦〩% (nice)

![image](https://user-images.githubusercontent.com/190571/182605965-774c58b4-558c-425e-9f72-7df5ec7356e4.png)

Note for later
```cpp
    QLocale locale = QLocale::system();
    QCOMPARE(locale.zeroDigit(), QStringView(u"\u3007"));
```

I'll see if I can later coax one of my virtual machines into using these numbers later.

**Comment by Docteh** [comment]
This actually works on both Windows 10 and Windows 11, take a look at what "Standard digits" is set to. 
![image](https://user-images.githubusercontent.com/190571/182732294-de2796c3-1426-43d0-8f0e-20ffc7cb7ac3.png)

**Comment by Nick-Hopps** [comment]
It's exactly what I've done, thx again.

**Comment by german77** [comment]
Can someone confirm is fixed on the PR artifacts of #8715

**Comment by Docteh** [comment]
PR artifact works fine. Just have to press the left shift to get keyboard into alphanumeric mode, and windows isn't too too bad to get around. 
![image](https://user-images.githubusercontent.com/190571/183540268-943c36bb-c700-4a34-9218-77f6cbf32b43.png)


I found an interesting quirk/bug when I use Regional format: `Chinese (Traditional, Hong Kong SAR) with the display language of English, yuzu ends up forcing language to English instead of allowing it to stay at `<System>`, happens on both Windows 10 and 11, with and without this PR. I even took a look at Qt's code for QTranslator::load and ¯\\\_(ツ)_/¯
